x-pack/plugins/timelines/public/components/hover_actions/actions/filter_for_value.tsx

```diff
@@ -35,11 +36,15 @@ const FilterForValueButton: React.FC<FilterForValueProps> = React.memo(
     showTooltip = false,
     value,
     dataViewId,
+    dataProvider,
   }) => {
+    const provider = Array.isArray(dataProvider) ? dataProvider[0] : dataProvider;
```

Comment:
same thing should be done for `filter_out_value` after we decide if this is a good way to solve this problem